addons/mail/static/src/discuss/core/public/welcome_page.xml

```diff
@@ -2,30 +2,46 @@
-                        <i class="fa" t-attf-class="{{ state.audioStream ? 'fa-microphone' : 'fa-microphone-slash' }}"/>
-                    </button>
-                    <button class="btn fa-stack align-self-end p-0 m-3 rounded-circle fs-1 shadow" t-attf-class="{{ state.videoStream ? 'btn-dark' : 'btn-danger' }}" t-on-click="onClickVideo">
-                        <i class="fa fa-camera"/>
-                    </button>
+        <h2 class="m-3 m-md-5" t-esc="store.companyName"/>
+        <div class="w-100 d-flex justify-content-center gap-5" t-att-class="{'flex-column': ui.isSmall}">
+            <div t-if="this.store.discuss_public_thread.default_display_mode === 'video_full_screen'" class="w-100 align-self-center" style="max-width: 640px;">
+                <div class="position-relative d-flex flex-column align-items-center" t-ref="root">
+                    <video class="shadow rounded-3 bg-dark object-fit-contain" t-attf-height="{{ui.isSmall ? 240 : 360}}" width="100%" autoplay="" t-ref="video"/>
+                    <p t-if="hasRtcSupport and !state.videoStream" class="position-absolute bottom-50 text-light">
+                        Camera is off
+                    </p>
+                    <p t-if="!hasRtcSupport" class="position-absolute bottom-50 text-light">
+                        Your browser does not support videoconference
+                    </p>
+                    <div class="position-absolute bottom-0">
+                        <button class="btn fa-stack align-self-end p-0 m-3 rounded-circle fs-1 shadow" t-attf-class="{{ state.audioStream ? 'btn-dark' : 'btn-danger' }}" t-on-click="onClickMic">
+                            <i class="fa" t-attf-class="{{ state.audioStream ? 'fa-microphone' : 'fa-microphone-slash' }}"/>
+                        </button>
+                        <button class="btn fa-stack align-self-end p-0 m-3 rounded-circle fs-1 shadow" t-attf-class="{{ state.videoStream ? 'btn-dark' : 'btn-danger' }}" t-on-click="onClickVideo">
+                            <i class="fa fa-camera"/>
+                        </button>
+                    </div>
+                    <audio autoplay="" t-ref="audio"/>
+                </div>
+                <div class="d-flex flex-column flex-md-row gap-2 text-muted mt-3">
+                    <div class="w-100">
+                        <i class="fa fa-microphone"></i> MICROPHONE
+                        <DeviceSelect kind="'audioinput'"/>
+                    </div>
+                    <div class="w-100">
+                        <i class="fa fa-volume-up"></i> OUTPUT
+                        <DeviceSelect kind="'audiooutput'"/>
+                    </div>
+                    <div class="w-100">
+                        <i class="fa fa-camera"></i> CAMERA
+                        <DeviceSelect kind="'videoinput'"/>
```

Comment:
Should use text-uppercase rather than let translators the responsibility to uppercase